src/main/java/com/firebolt/jdbc/client/account/response/FireboltEngineIdResponse.java

```diff
@@ -1,20 +1,35 @@
 package com.firebolt.jdbc.client.account.response;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
+import org.json.JSONObject;
 
-import lombok.Builder;
-import lombok.Value;
-
-@Value
-@Builder
 public class FireboltEngineIdResponse {
-	@JsonProperty("engine_id")
-	Engine engine;
+	private final Engine engine;
+
+    public FireboltEngineIdResponse(Engine engine) {
+        this.engine = engine;
+    }
+
+	FireboltEngineIdResponse(JSONObject json) {
+		this(new Engine(json.getJSONObject("engine_id")));
+	}
+
+	public Engine getEngine() {
+		return engine;
+	}
```

Comment:
Something's happening with indentation here and in many other places. Please use a linter.
![image](https://github.com/firebolt-db/jdbc/assets/93913847/3a8e4f4c-24ab-4f1e-a7ec-be57532b1ac6)

Reply:
fixed